README.md

```diff
@@ -9,26 +9,25 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/4fa2c17a-6385-4cc6-9919-e32c134175d9/deploy-status)](https://app.netlify.com/sites/blazorstatic/deploys)
 
 
-Embrace the capabilities of Blazor on .NET 8 to craft static websites.
```

Comment:
I would remove the "on .NET" part. There is no Blazor without dotnet.  Or maybe leave out Blazor and keep the .NET. Blazor is in the name and .NET is known better (?). Also it makes it shorter. 

Base on your decision, I will change the description.

Reply:
Yeah I can remove the .NET. What about "Embrace the power" instead of "capabilities"?

Reply:
yes, that is also something that was meant for .net 8 as the capabilities were not present prior..